psi4/driver/qcdb/libmintspointgrp.py

```diff
@@ -28,6 +28,7 @@
 
 from __future__ import absolute_import
 from __future__ import print_function
+from __future__ import division
```

Comment:
I think we just should stay away from division future just for clarity. Its a good rule of thumb to practice regardless of language semantics IMO.